src/os/query/areamanager.js

```diff
@@ -191,6 +192,26 @@ os.query.AreaManager.mapOriginalGeoms = function(area) {
 };
 
 
+/**
+ * @return {Array<!os.data.AreaNode>} Array of active areas
+ * @export
+ */
+os.query.AreaManager.prototype.activeAreas = function() {
+  var allAreas = this.getAll();
+  var activeAreas = [];
+  var index = 0;
+
+  allAreas.forEach((area) => {
+    if (area.get('shown')) {
+      activeAreas[index] = /** @type {!os.data.AreaNode} */ (new os.data.AreaNode(area));
```

Comment:
You should call `activeAreas.push(area)` here and get rid of the `index` variable altogether.

For future uses of the `forEach` call, it's worth checking the larger function signature for it and other built in JS API calls: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

If you do need the index when using this call at some point in the future, `forEach` passes it to the function you give it as the second argument i.e. `allAreas.forEach((ele, index, arr) => {...}`.